### PR TITLE
Remove useless index seek

### PR DESCRIPTION
**Pull request by shenli**
We only need get duplicate row handle when running insert on duplicate update or replace stmt.
@coocood @zimulala @hanfei19910905 @tiancaiamao

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
PTAL @coocood

File: table/tables/index.go [hunk 1]
```diff
@@ -158,24 +158,27 @@ func (c *index) GenIndexKey(indexedValues []types.Datum, h int64) (key []byte, d
 
 // Create creates a new entry in the kvIndex data.
 // If the index is unique and there is an existing entry with the same key, Create will return ErrKeyExists.
```
Comment (coocood) [inline table/tables/index.go]: Add comment about returns existing handle.

Comment (shenli) [inline table/tables/index.go]: PTAL @coocood

**Comment by coocood** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

**Pull request merged**